src/autodiscovery.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
yeah lets not disable this for entire files. Do it per instance so that we can see the sheer quantity of the issue.

Reply:
@t3chguy AFAIK this one's only an example, that's why it hasn't been touched much. It probably can be left as .js file though.

Reply:
> AFAIK this one's only an example

Sorry what's an example?

![image](https://user-images.githubusercontent.com/2403652/129236802-ddd4e73a-10e2-497f-9613-34087ba8c901.png)